src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -251,6 +251,19 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
+    /**
+     * Converts {@code date} to UTC according to the {@code timeZone} offset and formats it according to ISO8601.
+     */
+    public static String convertAndFormatDateToIso8601Utc(Date date, double timeZone) {
+        if (date == null) {
+            return "";
+        }
+        // Passing negative timeZone to convert to UTC.
+        Calendar calInUtc = convertToUserTimeZone(dateToCalendar(date), -timeZone);
```

Comment:
@whipermr5 While I like the idea, the `convertLocalDateToUtc` method _is_ deprecated whereas this method is super useful and can live on even after `convertLocalDateToUtc` is removed. Given this, would it still be a good idea to use it?